src/SvgIcon/SvgIcon.js

```diff
@@ -0,0 +1,72 @@
+// @flow weak
+
+import React, { PropTypes } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import ClassNames from 'classnames';
+import { easing } from '../styles/transitions';
+
+export const styleSheet = createStyleSheet('SvgIcon', (theme) => {
+  return {
+    SvgIcon: {
+      display: 'inline-block',
+      fill: 'currentColor',
+      height: 24,
+      width: 24,
+      userSelect: 'none',
+      transition: theme.transitions.create('fill', '200ms', null, easing.easeOut),
```

Comment:
Why not using the default `easeInOut` easing curve?
https://material.google.com/motion/duration-easing.html#duration-easing-natural-easing-curves